src/actions/actionMenu.tsx

```diff
@@ -45,3 +47,19 @@ export const actionToggleEditMenu = register({
     />
   ),
 });
+
+export const actionFullScreen = register({
+  name: "toggleFullScreen",
+  perform: () => {
+    if (!isFullScreen()) {
+      allowFullScreen();
+    }
+    if (isFullScreen()) {
+      exitFullScreen();
+    }
+    return {
+      commitToHistory: false,
+    };
+  },
+  keyTest: (event) => event.key === KEYS.F_KEY,
```

Comment:
can we use a key code or something so it won't matter if it's `f` or `F`.. it doesn't work with capital :)